packages/experiments/src/components/TilesList/examples/TilesList.Media.Example.tsx

```diff
@@ -125,7 +146,7 @@ export class TilesListMediaExample extends React.Component<{}, ITilesListMediaEx
       background: (
         <img
           className={TilesListExampleStyles.tileImage}
-          src={`//placehold.it/${Math.round(backgroundSize.width)}x${Math.round(backgroundSize.height)}`}
+          src={`http://placekitten.com/${Math.round(backgroundSize.width)}/${Math.round(backgroundSize.height)}`}
```

Comment:
We have been instructed not to use `placekitten` due to attribution issues.

Reply:
Haha, okay I'll change it back. It was a little difficult to get a good feel for how it would really look with just grey boxes.